tests/cases/text/control_keys_input.slint

```diff
@@ -0,0 +1,39 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+TestCase := Window {
+    width: 100phx;
+    height: 100phx;
+    ti := TextInput { 
+    }
+
+    property <bool> input_focused: ti.has_focus;
+    property <string> text <=> ti.text;
+}
+
+/*
+```rust
+
+const SHIFT_CODE: char = '\u{0010}';
+const SHIFTR_CODE: char = '\u{0015}';
+const ALT_CODE: char = '\u{0012}';
+const ALTGR_CODE: char = '\u{0013}';
+const CONTROL_CODE: char = '\u{0011}';
+const CONTROLR_CODE: char = '\u{0016}';
+const META_CODE: char = '\u{00E0}';
+const METAR_CODE: char = '\u{00E1}';
```

Comment:
(Note: there is public API to get these codes now.  (but we don't need to use them in the test))